deployment.yaml

```
@@ -0,0 +1,53 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+   name: module-14
+spec:
+   selector:
+     matchLabels:
+       app: module-14
+   replicas: 1
+   template:
+     metadata:
+        labels:
+          app: module-14
+     spec:
+        containers:
+          - name: image 
+            image: todo-app:prod2
+            env:
+            - name: SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                 name: secret
+                 key: secret_key
+            - name: client_id
+              valueFrom:
+                secretKeyRef:
+                  name: secret
+                  key: client_id
+            - name: client_secret
+              valueFrom:
+                secretKeyRef:
+                  name: secret
+                  key: client_secret
+            - name: mongo_client
+              valueFrom:
+                secretKeyRef:
+                  name: secret
+                  key: mongo_client
+            - name: LOGGLY_TOKEN
+              valueFrom:
+                secretKeyRef:
+                 name: secret
+                 key: LOGGLY_TOKEN
+            - name: LOG_LEVEL
+              valueFrom:
+                secretKeyRef:
+                 name: secret
+                 key: LOG_LEVEL
```

This value isn't sensitive, so we could set it directly, like:
```suggestion
            - name: LOG_LEVEL
              value: INFO
```

---

done